ocaml/xapi/xapi_pif.ml

```diff
@@ -753,15 +757,28 @@ let reconfigure_ipv6 ~__context ~self ~mode ~iPv6 ~gateway ~dNS =
   Db.PIF.set_ipv6_configuration_mode ~__context ~self ~value:mode ;
   Db.PIF.set_ipv6_gateway ~__context ~self ~value:gateway ;
   Db.PIF.set_IPv6 ~__context ~self ~value:[iPv6] ;
-  (* Keep IPv4 DNS entries *)
-  let pif_dns =
+  (* Only keep existing IPv4 entries *)
+  let pif_ipv4_dns =
```

Comment:
A different name for a similar purpose here.

Reply:
what do you think about:
`<family>_dNS` for the DNS inputed to the reconfigure function and `pif_<family>_dNS` for the DNS already present in the PIF DNS field.
Where family is either `ipv6` or `ipv4`